README.md

```diff
@@ -1,68 +1,154 @@
 
-Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages.
+```html
+<script type="module">
+  import * as LR from 'https://cdn.jsdelivr.net/npm/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks.min.js';
+  
+  LR.registerBlocks(LR);
+</script>
+```
 
-Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
+* Start using `Blocks` in your application markup (don't forget to specify `@{{PACKAGE_VERSION}}` with the [latest one](https://github.com/uploadcare/blocks/releases)):
 
-## Supported browsers
+```html
+<lr-file-uploader-regular
+  css-src="https://cdn.jsdelivr.net/npm/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-regular.min.css"
+  class="my-uploader"
+>
+</lr-file-uploader-regular>
+```
 
-Blocks are supported and tested in all major modern desktop and mobile browsers:
+* All configurations and customization in `Blocks` are managed from CSS code. Sign up to [Uploadcare](https://app.uploadcare.com/accounts/signup/) and get a Public API key in [Uploadcare project's dashboard](https://app.uploadcare.com/projects/-/api-keys/). Then add a style sheet to your project and replace `'YOUR_PUBLIC_KEY'` with your own public key:
 
-- Chrome
-- Firefox
-- Safari
-- Edge
-- Opera
+```css
+.my-uploader {
+  --ctx-name: 'my-uploader';
+  --cfg-pubkey: 'YOUR_PUBLIC_KEY';
+}
+```
 
-Internet Explorer is outdated and not supported [anymore](https://uploadcare.com/blog/uploadcare-stops-internet-explorer-support/).
+### Using NPM
 
-## Frameworks support
+* Install Blocks package: `npm i --save-exact @uploadcare/blocks`
+* Connect `Blocks` from your script file:
+```js
+import * as LR from '@uploadcare/blocks';
 
-Blocks are a framework-agnostic solution, so you can use it with any runtime you like. We have prepared _examples_ for the most popular frameworks.
+LR.registerBlocks(LR);
+```
+* Start using `Blocks` in your application markup and replace `@{{PACKAGE_VERSION}}` with the [latest version](https://github.com/uploadcare/blocks/releases) of the package:
+```html
+<lr-file-uploader-inline
+        css-src="https://cdn.jsdelivr.net/npm/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-regular.min.css"
+        class="my-uploader"
+>
+</lr-file-uploader-inline>
+```
+
+### Using File Uploading Features
+
+All configurations in `Blocks` are managed from CSS code. 
+* Sign up to [Uploadcare](https://app.uploadcare.com/accounts/signup/).
+* Get a Public API key in [Uploadcare project's dashboard](https://app.uploadcare.com/projects/-/api-keys/). 
+* Add a style sheet to your project and replace `'YOUR_PUBLIC_KEY'` with your own public key:
+
+```css
+.my-uploader {
+  --ctx-name: 'my-uploader';
+  --cfg-pubkey: 'YOUR_PUBLIC_KEY';
+}
+```
```

Comment:
Let's add a link to the https://uploadcare.com/docs/file-uploader/configuration/